src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/DoctrineTransport.php

```diff
@@ -21,6 +21,9 @@
 use Symfony\Component\Messenger\Transport\SetupableTransportInterface;
 use Symfony\Component\Messenger\Transport\TransportInterface;
 
+if (!class_exists(DbalConnection::class) || !class_exists(\Doctrine\Persistence\ConnectionRegistry::class)) {
+    throw new \LogicException('doctrine/dbal and doctrine/persistence are required to use DoctrineTranspart. Please run "composer require doctrine/dbal doctrine/persistence".');
```

Comment:
~DoctrineTranspart~ DoctrineTransport

Reply:
I just removed this: we don't need it, my bad.